web/src/features/dashboard/components/LeftAlignedCell.tsx

```diff
@@ -0,0 +1,10 @@
+import { cn } from "@/src/utils/tailwind";
+import { type ReactNode } from "react";
+
+export const LeftAlignedCell = ({
```

Comment:
i'd inline this, a simple text-left is easier to understand on a div than in a component in another file (that's why we use tailwind)

Reply:
just saw the right aligned cell, i'd also remove it as this makes our code more convoluted